### PR TITLE
docs: document "listOn" properties (and remove "canPurchaseNfts")

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: docs/adding-new-dapps.md [hunk 1]
```diff
@@ -17,14 +17,15 @@ There are some fields that need to be translated into different languages, and t
...
+| `id`            | A unique identifier for the Dapp. The current pattern is to use a lower-kebab-case string.                                                                              |
+| `categoryId`    | The category id under which the Dapp should show up. This will be deprecated soon, it will be replaced by `categories`.                                                 |
+| `categories`    | An array of category ids under which the Dapp should show up.                                                                                                           |
+| `url`           | The URL to open when the Dapp is selected. Can also be a deep link to open in-app.                                                                                      |
+| `listOnAndroid` | Whether to list the Dapp on Android devices. Your Dapp must be compliant with the [Play Store Content Policy](https://play.google.com/about/developer-content-policy/). |
+| `listOnIos`     | Whether to list the Dapp on iOS devices. Your Dapp must be compliant with the [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/).  |
```
Comment (jeanregisser) [inline docs/adding-new-dapps.md:28]: Should we keep the example of purchasing NFTs?
just to give a more concrete use case.

**Review by jeanregisser — APPROVED** [review]
🙌

File: docs/adding-new-dapps.md (hunk 1 above)
Comment (silasbw) [reply docs/adding-new-dapps.md:28]: I intentionally removed it: i was worried readers might focus on that one specific example and there isn't a lot of space to provide a more exhaustive answer.  we could add a FAQ or something.  E.g., "What are common issues with being listed on iOS?"

**Comment by silasbw** [comment]
@dievazqu yep -- already updated :+1:

File: docs/adding-new-dapps.md (hunk 1 above)
Comment (silasbw) [reply docs/adding-new-dapps.md:28]: added a faq, fyi

**Pull request merged**